.github/workflows/triage-issues.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow uses the following github action to automate
+# management of stale issues and prs in this repo:
+# https://github.com/marketplace/actions/close-stale-issues
+
+name: Close stale issues and PRs
+
+on: 
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 0'
+  
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4.0.0
+        with:
+          days-before-issue-stale: 30
+          days-before-issue-close: 10
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
+          stale-issue-label: stale-no-activity
```

Comment:
If we use the term "auto-triage", consistently having the term in the labels may be easier to understand like this:
* stale-issue-label: auto-triage-stale
* exempt-issue-labels: auto-triage-skip

What do you think?

Reply:
I like that - I will add those updates.